include/circt/Dialect/SV/SVStatements.td

```diff
@@ -474,3 +481,27 @@ def CoverOp : SVOp<"cover", [ProceduralOp]> {
   let results = (outs);
   let assemblyFormat = "attr-dict $property `:` type($property)";
 }
+
+def BindOp : SVOp<"bind", [IsolatedFromAbove, HasParent<"mlir::ModuleOp">]> {
+  let summary = "indirect instantiation statement";
+  let description = [{
+    Indirectly instantiate a module in the context of another module.  This
+    implements all-instance module binding in system verilog.  This will
+    generate and validate ".*" name binding.
```

Comment:
Please mention that this is in section 23.11 of the spec, it took me forever to find it :-)